packages/components/src/Menu/MenuItem.test.tsx

```diff
@@ -26,24 +26,32 @@
 
 import 'jest-styled-components'
 import React from 'react'
-import { assertSnapshot } from '@looker/components-test-utils'
+import { renderWithTheme } from '@looker/components-test-utils'
 
 import { MenuItem } from './MenuItem'
 
-test('MenuItem', () => {
-  assertSnapshot(<MenuItem>who!</MenuItem>)
-})
-test('MenuItem - icon', () => {
-  assertSnapshot(<MenuItem icon="Beaker">who!</MenuItem>)
+test('MenuItem renders', () => {
+  const { getByText } = renderWithTheme(<MenuItem>who!</MenuItem>)
+  expect(getByText('who!')).toBeVisible()
 })
+
 test('MenuItem - detail', () => {
-  assertSnapshot(<MenuItem detail="Is an excellent question">who!</MenuItem>)
+  const { getByText } = renderWithTheme(
+    <MenuItem detail="Is an excellent question">who!</MenuItem>
+  )
+  expect(getByText('Is an excellent question')).toBeVisible()
```

Comment:
We've been switching to the convention of importing `screen` and not de-structuring the helpers from the renderWithTheme.

```suggestion
  renderWithTheme(
    <MenuItem detail="Is an excellent question">who!</MenuItem>
  )
  expect(screen.getByText('Is an excellent question')).toBeVisible()
```